frontend/text.js

```diff
@@ -27,6 +27,18 @@ class Text {
     return this.elems[index].elemId
   }
 
+  /**
+   * Returns a cursor that points to a specific point in the text.
+   * For now, represented by a plain object.
+   */
+  getCursorAt (index) {
+    return {
+      // todo: are there any points in the lifecycle where the Text object doesn't have an ID?
```

Comment:
@ept I see some [other places](https://github.com/automerge/automerge/blob/3e72b69bafb44d6625496b9d956fdad181530006/frontend/text.js#L116) that handle the case where the object ID is not set.

Would it be sensible to add a similar check here and forbid getting a cursor if there's not an object ID? I don't yet understand when/how that case occurs.